module_enumerable.rb

```diff
@@ -0,0 +1,23 @@
+module MyEnumerable
+  def all?
+    each do |item|
+      return false unless yield item
+    end
+    true
+  end
+
+  def any?
+    each do |item|
+      return true if yield item
+    end
+    false
+  end
+
+  def filter?(&block)
```

Comment:
- [ ] I think it would be more appropriate to name this method just `filter`, instead of `filter?` as you have in the project requirements. Plus the method does not return `true` or `false`, so adding `?` does not suit the code.

  ![image](https://user-images.githubusercontent.com/36542986/224023095-6e3a0f0c-9509-4abf-afc1-3e66be9c1cce.png)